src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,55 @@
+    <DefineConstants Condition="'$(IsPlaceholderTargetFramework)' != 'true'">$(DefineConstants);NETSTANDARD</DefineConstants>
+    <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsPlaceholderTargetFramework)' == 'true'">
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
+    <Compile Include="$(CommonPath)\System\Buffers\Text\FormattingHelpers.CountDigits.cs" Link="Common\System\Buffers\Text\FormattingHelpers.CountDigits.cs" />
+    <Compile Include="$(CommonPath)\System\MutableDecimal.cs" Link="Common\System\MutableDecimal.cs" />
+    <Compile Include="$(CommonPath)\System\Numerics\Hashing\HashHelpers.cs" Link="Common\System\Numerics\Hashing\HashHelpers.cs" />
+    <Compile Include="$(CommonPath)\System\Buffers\IMemoryOwner.cs" Link="Common\System\Buffers\IMemoryOwner.cs" />
+    <Compile Include="$(CommonPath)\System\Buffers\IPinnable.cs" Link="Common\System\Buffers\IPinnable.cs" />
+    <Compile Include="$(CommonPath)\System\Buffers\MemoryHandle.cs" Link="Common\System\Buffers\MemoryHandle.cs" />
+    <Compile Include="$(CommonPath)\System\Buffers\MemoryManager.cs" Link="Common\System\Buffers\MemoryManager.cs" />
+    <Compile Include="$(CommonPath)\System\Memory.cs" Link="Common\System\Memory.cs" />
+    <Compile Include="$(CommonPath)\System\MemoryDebugView.cs" Link="Common\System\MemoryDebugView.cs" />
+    <Compile Include="$(CommonPath)\System\MemoryExtensions.cs" Link="Common\System\MemoryExtensions.cs" />
+    <Compile Include="$(CommonPath)\System\ReadOnlyMemory.cs" Link="Common\System\ReadOnlyMemory.cs" />
+    <Compile Include="$(CommonPath)\System\ReadOnlySpan.cs" Link="Common\System\ReadOnlySpan.cs" />
+    <Compile Include="$(CommonPath)\System\Runtime\InteropServices\MemoryMarshal.cs" Link="Common\System\Runtime\InteropServices\MemoryMarshal.cs" />
+    <Compile Include="$(CommonPath)\System\Span.cs" Link="Common\System\Span.cs" />
+    <Compile Include="$(CommonPath)\System\SpanDebugView.cs" Link="Common\System\SpanDebugView.cs" />
+    <Compile Include="$(CommonPath)\System\SpanHelpers.BinarySearch.cs" Link="Common\System\SpanHelpers.BinarySearch.cs" />
+    <Compile Include="$(CommonPath)\System\SpanHelpers.Byte.cs" Link="Common\System\SpanHelpers.Byte.cs" />
+    <Compile Include="$(CommonPath)\System\SpanHelpers.Char.cs" Link="Common\System\SpanHelpers.Char.cs" />
+    <Compile Include="$(CommonPath)\System\SpanHelpers.T.cs" Link="Common\System\SpanHelpers.T.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
+    <ProjectReference Include="..\..\System.Buffers\src\System.Buffers.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
+    <PackageReference Include="System.Numerics.Vectors" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
+  </ItemGroup>
```

Comment:
nit: merge the item groups together

Reply:
I can do that, but I thought we preferred to have different items in their own item groups.

Reply:
Not really. In some cases it makes sense but here it just adds unnecessary code (duplicate conditions).